db/migrate/20160308155249_add_online_application_tables.rb

```diff
@@ -0,0 +1,48 @@
+class AddOnlineApplicationTables < ActiveRecord::Migration
+  def change
+    create_table :online_failures do |t|
+      t.text :received_data, null: false
+      t.timestamps null: false
+    end
+
+    create_table :online_applications do |t|
+      t.boolean :married, null: false
+      t.boolean :threshold_exceeded, null: false
+      t.boolean :benefits, null: false
+      t.integer :children, null: false
+      t.integer :income
+
+      t.boolean :refund, null: false
+      t.string :date_fee_paid
```

Comment:
My fault, I never updated the source, but this (and a couple of others) should be t.date?